tensorflow/tools/dist_test/README.md

```diff
@@ -1,6 +1,6 @@
 # Testing Distributed Runtime in TensorFlow
-This folder containers tools and test suites for the GRPC-based distributed
-runtime in TensorFlow.
+This folder contains tools and test suites for the distributed runtime
```

Comment:
`runtime` -> `runtimes`. Please explain that there are two types of distributed runtimes: GRPC-based and Allreduce-based. Please also add a reference to the Allreduce-based model (the Horovod paper is fine).

Reply:
Sure. I added GRPC-based and Allreduce-based to that line, and added a reference to Horovod paper in "Create allreduce-based Tensorflow k8s deployment" section.